.gitignore

```diff
@@ -11,6 +12,7 @@ dist/
 # VitePress
 docs/.vitepress/cache/
 docs/.vitepress/temp/
+.vitepress/
```

Comment:
Why?

Reply:
I think my IDE or some process put this here-- there was a .vitepress/ folder at the root, with only a `caches` or something like that folder inside. To minimize changes here i just deleted that folder, removed this line so should be all set!